src/Mod/Sketcher/Gui/DrawSketchHandlerRotate.h

```diff
@@ -492,9 +492,9 @@ void DSHRotateController::configureToolWidget()
             WCheckbox::FirstBox,
             QApplication::translate(
                 "TaskSketcherTool_c1_offset",
-                "This concerns the datum constraints like distances. If you activate Clone, "
-                "then the tool will copy the datum. Else it will try to replace them with "
-                "equalities between the initial geometries and the new copies."));
+                "This concerns the datum constraints like distances.\nIf you activate Clone, "
+                "then the tool will copy the datum.\nElse it will try to replace them with "
+                "equal constraints."));

```

Comment:
Ideally it is better to keep the html out of the translated texts. So you could do like this : 
```
        toolWidget->setCheckboxToolTip(
            WCheckbox::FirstBox,
            QString::fromLatin1("<p>")
            + QApplication::translate( "TaskSketcherTool_c1_offset",
                "This concerns the datum constraints like distances.If you activate Clone, "
                "then the tool will copy the datum. Else it will try to replace them with "
                "equalities between the initial geometries and the new copies.") 
            + QString::fromLatin1("</p>"));
```

Reply:
@WandererFan would be great if you could place the html outside the translate text as I am sometimes struggling with the TechDraw translations :)